src/components/service/dateFormatting.js

```diff
@@ -0,0 +1,19 @@
+export default function formattingDate(date) {
+  const formatting = new Date(date);
```

Comment:
`new Date(date).toLocaleString('en-GB', { day: 'numeric', month: 'long', year: 'numeric' })` seems to be the same